src/modules/keyboardmanager/dll/KeyboardEventHandlers.cpp

```diff
@@ -98,6 +99,12 @@ namespace KeyboardEventHandlers
                 UINT res = ii.SendVirtualInput(key_count, keyEventList, sizeof(INPUT));
                 delete[] keyEventList;
 
+                // Log telemetry event when the key remap is invoked
+                if (data->wParam == WM_KEYDOWN || data->wParam == WM_SYSKEYDOWN)
+                {
+                    Trace::KeyRemapInvoked(remapToKey);
```

Comment:
@arjunbalgovind shouldn't this go inside the below if statement.

Reply:
The if block below was specifically for the Caps Lock IME issues, so I wanted to keep the unrelated logic separate from each other

Reply:
Removed the redundant if check